HOWTO.md

```diff
@@ -734,3 +736,31 @@ Options:
 
 Please note that in this mode there are some environment variables that are
 still required to be set by the user (`DI_SIGN_KEY_PATH`, `DI_HMAC_KEY_PATH`).
+
+## How to use Features
+  ### How to use `device_specific_store_driver` feature
+  - Using this feaure user can choose to apply different serviceinfo settings to different devices.
```

Comment:
Since this is a general explanation of how the feature works I would put it outside the `-` as a general explanation.

typo: "... this feature the user"  "settings on different devices."